internal/alerts/example-alert-template.json

```diff
@@ -0,0 +1,296 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {},
+    "resources": [
+        {
+            "name": "containerinsights_dev_mac_eus_alerts",
+            "type": "Microsoft.AlertsManagement/prometheusRuleGroups",
+            "apiVersion": "2023-03-01",
+            "location": "eastus",
+            "properties": {
+                "description": "rule group for cluster ci-dev-aks-mac-eus in MAC: ci-dev-aks-eus-mac",
+                "scopes": [
+                    "/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-dev-aks-mac-eus-rg/providers/microsoft.monitor/accounts/ci-dev-aks-eus-mac",
+                    "/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-dev-aks-mac-eus-rg/providers/Microsoft.ContainerService/managedClusters/ci-dev-aks-mac-eus"
+                ],
+                "rules": [
+                    {
+                        "alert": "Amd64 metric missing in cluster ci-dev-aks-mac-eus",
+                        "expression": "absent(node_uname_info{machine=\"x86_64\"}) == 1 or node_uname_info{machine=\"x86_64\"} == 0",
+                        "for": "PT3M",
+                        "labels": {
+                            "cluster": "ci-dev-aks-mac-eus"
```

Comment:
please move this to clusterName scope per rule group.

Reply:
Thanks Vishwa! I have updated this now.